librz/analysis/type_pdb.c

```diff
@@ -28,237 +49,634 @@ static char *create_type_name_from_offset(ut64 offset) {
 	return str;
 }
 
+static RzType *parse_type_array(const RzTypeDB *typedb, STypeInfo *type) {
```

Comment:
Please add `rz_return_val_if_fail(type, NULL)` for all these functions to avoid programmatic mistakes in using those